package.json

```diff
@@ -61,7 +61,8 @@
     "mini-css-extract-plugin": "^1.6.2",
     "nth-check": "^2.0.1",
     "react": "^17.0.2",
-    "typescript": "^4.5.5"
+    "typescript": "^4.5.5",
+    "webpack-dev-server": "^4.7.4"
```

Comment:
should this be added in the ssr package?

Reply:
I thought you need to add it in both. It is in the SSR package

Reply:
In theory, packages should only be in the root package.json if the dependency is not for a specific package and is required to run the repo.

Reply:
It seems that we'll still download the old version if I don't place it in the root package.json